pkgs/default.nix

```diff
@@ -14,6 +14,7 @@
   bazel_ot = pkgs.callPackage ./bazel_ot {};
   llvm_cheriot = pkgs.callPackage ./llvm_cheriot.nix {};
   ibex-cosim = pkgs.callPackage ./ibex-cosim.nix {};
+  xmake = pkgs.callPackage ./xmake.nix {};
```

Comment:
I'd suggest to use `import` instead of `callPackage` if the package is itself the override. This case if some body wants to further override, it will use the original override provided by pkgs.xmake instead of the overriden one which is less tuneable.